includes/bookmark.inc

```diff
@@ -841,6 +841,10 @@ abstract class Bookmark implements BookmarkInterface {
         $params[] = $form_state['values']['bookmarks']['fieldset']['format_opts']['export_styles'][$key];
```

Comment:
Technically it wouldn't be needed. However, any existing implementations in the wild would have to be updated to reflect this change in parameter. Went for the additive route to mitigate this.